app/views/chargeback/_cb_rate_edit.html.haml

```diff
@@ -64,3 +65,6 @@
               /if the rate detail have a metric associated, display an options field with per_unit selected
               %td
                 = select_tag("per_unit_#{i}", options_for_select(measure.measures, r.per_unit), "data-miq_observe" => {:url => url}.to_json)
+          /Show the code of the currency selected by the user
+          %td{:id => "column_currency_#{i}"}
+            = r.detail_currency.code
```

Comment:
Is this hitting the database to bring back the detail_currency for each `@sb[:rate_details]`?  If so, this is an N+1 query.
@h-kataria @martinpovolny do you have recommendations to remove some of this logic from the view here?